pithos/plugin.py

```diff
@@ -18,16 +18,23 @@
 import os
 from gi.repository import (
     GLib,
-    Gio
+    Gio,
+    GObject
 )
 
 
-class PithosPlugin:
+class PithosPlugin(GObject.Object):
+    __gtype_name__ = 'PithosPlugin'
+    __gsignals__ = {
```

Comment:
I can do that. I really don't want to bind it to anything ATM though. Prob just connect to the notify signal.